l10n_ch_invoice_reports/data/reports.xml

```diff
@@ -13,17 +13,21 @@
         <field name="print_report_name">'Invoice-ISR-%s' % object.name</field>
         <field name="attachment_use" eval="False" />
         <field name="attachment" />
+        <field name="binding_model_id" ref="account.model_account_move" />
+        <field name="binding_type">report</field>
```

Comment:
The default value for `binding_type` is already `report` this line is not necessary.
https://github.com/odoo/odoo/blob/14.0/odoo/addons/base/models/ir_actions_report.py#L92